charters/ingestion-service/charter.md

```diff
@@ -0,0 +1,177 @@
+* Provide services to communicate to submitters any errors that have encountered during the submission process, irrespective of the method used (e.g. the user interface, a spreadsheet, or some other programmatic client). All submission interfaces must communicate enough information to allow a contributor to review any issues with a submission through the single endorsed user interface, and if possible to take action to repair the erroneous contents
+* Develop and maintain services that allow submission of metadata supplied in the form of a canonical spreadsheet format
+* Ensure spreadsheet submission services support the submission of spreadsheets that comprehensively capture metadata describing all aspects of an experiment that may be required for downstream analysis, even if this means the metadata *does not currently validate* against endorsed community standards
+* Provide services to allow spreadsheets to be re-submitted in order to make updates to an existing submission. These services can place additional restrictions on spreadsheets (for example, they may require the addition of a UUID field in order to identify the target records being updated)
+
+### Core capabilities
+
+* Develop and maintain a RESTful Ingest API supporting the collection of experimental metadata via submissions
+* Provide validation services that supply detailed reports, available through this API, on the quality of submitted data with respect to the endorsed community standards delivered by the metadata team
+* Provide state tracking services that supply information about the current state of any given submission, allowing submitters to see where in the ingestion process their data is currently
+* Provide services for connecting metadata describing experimentally generated data files, and the locations of those data files in the cloud
+* Provide services that syntactically validate experimentally generated data files, covering commonly used and endorsed community data formats (e.g. FASTQ)
+* Provide services that allow data contributors to confirm their submissions, prompting storage data and metadata in the DCP data store
+* Provide services for restricting access to only those submissions and their contents for which a user has either created themselves or been explicitly granted permission to view
+
+### Supporting tools
+
+* Provide tools that use reference (latest) metadata schemas, defined by the metadata team, to produce template spreadsheets in a canonical spreadsheet format, such that data contributors can easily complete this spreadsheet to capture their metadata
+* Provide tools that allow data contributors (normally, data wranglers) to generate customised spreadsheets that target specific experiment types or data types based on a small number of screening criteria
```

Comment:
> (normally, data wranglers)

In the near future, this might be mostly data wranglers, but I don't think it's limited to data wranglers. The goal is for any data contributor to come in to the Ingest Service and be given tools to generate their won customized template spreadsheet. I would remove this phrase.